src/provisioner/cluster.py

```diff
@@ -86,12 +87,24 @@ def _get_kvs(self, prefix, node):
             kvs.append((prefix, node))
         return kvs
 
+    @staticmethod
+    def _add_comp_version(components):
+        """Add rpm version key for each component."""
+        updates_list = []
+        cortx_release_info = CortxRelease()
```

Comment:
instead of instantiating CortxRelease multiple times, it should be supplied from the top class.

1. CortxProvisioner.config_apply() - Instantiate CortxRelease in this class.
2. Pass the value down as: 
    CortxProvisioner.config_apply_cluster(cortx_config_store, cortx_release)
    CortxProvisioner.config_apply_cortx(cortx_config_store, cortx_release)
3. In config_apply_cortx(), Pass this down. 
    cs = CortxConfig(cortx_config, cortx_release)
4. In config_apply_cluster() pass this down. 
    cs = CortxCluster(nodes, CortxRelease)

Reply:
Done